slsc-face-archive.md

```diff
@@ -0,0 +1,20 @@
+# Project SLSC Photo Archive Face Tagging
+
+## Client Profile
+
+Youth Exploring Science (YES) program is created to help underserved students in St. Louis area. Students participate in YES program to gain skills in Science, Technology, Engineering, Arts and Mathematics fields. High school students will gain real-world skills, career readiness training through YES program. Students in the YES program also promote the importance of STEAM-related careers.
+
+## Project Brief
+
+The SLSC Photo Archive Face project is to Identify and Tag people from the images client or user uploads, next once the detected faces tagging is completed we need to store the organized result in a database readily available to access, the client with the help of photo tagging interface would like to sort photos with respect to student which helps them to keep track of each students progress. In the Youth Exploring Science program, there are thousands of photos of students. Most of the photos are on Dropbox, Flicker and Google Photos. Currently the photos are organized by event and date. We should be able to find a way where we could be able to sort photos by student names and face which makes easy to collect information of each student without looking up the dates and events.
+
+## Functional Requirements
+
+* Data collection (All the photos with their name tags),
+* Preprocess data
+* Train a model or adapt a pre-trained model to fit our data
```

Comment:
Since we would be dealing with facial recognition I believe a pre-trained model would best suit us rather than writing and training a model from scratch

Reply:
noted;